src/PythonMigrationViewExtension/IronPythonInfoDialog.xaml.cs

```diff
@@ -12,8 +16,50 @@ internal IronPythonInfoDialog(PythonMigrationViewExtension viewModel)
         {
             this.ViewModel = viewModel;
             InitializeComponent();
+            SetPythonDepedenciesData();
         }
 
+        private void SetPythonDepedenciesData()
+        {
+            // Filter the Custom Nodes that have a direct dependency on IronPython2. 
+            var customNodesWithPythonDependencies = GraphPythonDependencies.CustomNodePythonDependency
+                                                            .Where(x => x.Value.Equals(GraphPythonDependencies.CNPythonDependency.DirectDependency));
+
+            if (customNodesWithPythonDependencies.Any())
+            {
+                foreach (var entry in customNodesWithPythonDependencies)
+                {
+                    ViewModel.CustomNodeManager.TryGetNodeInfo(entry.Key, out CustomNodeInfo customNodeInfo);
+
+                    CustomNodeItem customNodeItem = new CustomNodeItem
+                    {
+                        Name = customNodeInfo.Name,
+                        FunctionId = customNodeInfo.FunctionId,
+                        PackageName = customNodeInfo.PackageInfo != null ? customNodeInfo.PackageInfo.Name : "Definitions",
+                    };
+
+                    if (customNodeInfo.PackageInfo != null)
+                    {
+                        PackagedCustomNodesContainingPython.Items.Add(customNodeItem);
```

Comment:
is there a reason to implement this manually instead of using a binding?

Reply:
also if you do decide to use a binding remember to use an observable collection.

Reply:
As this list is static and doesn't have much functionality added to it, we decided to keep it simple.